validator/project_validator.go

```diff
@@ -24,6 +24,7 @@ const (
 	Error ValidationErrorLevel = iota
 	Warning
 	unauthorizedCharacters = "|"
+	maxTimes               = 50
```

Comment:
We could remove it but I think it's generally good practice to cap something that could go otherwise go too far. I'm not particularly attached to that reasoning though.